game.cpp

```diff
@@ -351,49 +355,55 @@ bool Game::RunFrame(double dt) {
     }
 
     // Render stuff
-    
+    IRender* renderer = GetRenderer();
     // ImGUI stuff goes into GL default FBO
-    m_Renderer->RenderBegin();
+    renderer->RenderBegin();
 
     ImGui::ShowDemoWindow();
 
     // Main 3D: This is where all the 3D rendering happens (in its own FBO)
     {
-        m_Renderer->Begin3D();
+        renderer->Begin3D();
         
         // Draw Debug Line for player veloctiy vector
         Line velocityDebugLine = {
             Vertex(ec.center), Vertex(ec.center + 200.0f * m_Player.velocity)
         };
         velocityDebugLine.a.color = glm::vec4(1.0f, 1.0f, 0.0f, 1.0f);
         velocityDebugLine.b.color = velocityDebugLine.a.color;
-        m_Renderer->ImDrawLines(velocityDebugLine.vertices, 2, false);
+        renderer->ImDrawLines(velocityDebugLine.vertices, 2, false);
 
+        // Render World Geometry (Batched triangles)
+        //renderer->SetActiveCamera(&m_FollowCamera);
+        //renderer->DrawWorldTris();
+
+#if 0
         // Render World geometry
-        m_Renderer->ImDrawTriPlanes(m_World.m_TriPlanes.data(), 
-                                    m_World.m_TriPlanes.size(), 
-                                    true,
-                                    DRAW_MODE_SOLID);
+        renderer->ImDrawMapTris(m_World.m_Tris.data(), 
+                                  m_World.m_Tris.size(), 
+                                  true,
+                                  DRAW_MODE_SOLID);
```

Comment:
noob question: why do we need to register the tris with the renderer and afterwards pass them again into the draw call? couldn't we just save the tris in the renderer for static data?

Reply:
Not a noob question. It was old code (that is not compiled due to `#if 0`). Thanks for spotting this! Removed.